### PR TITLE
Bug fixes - switch between vod and webcast

### PR DESCRIPTION
**Pull request by ibirman-vbrick**
Also fix reference to vbrickEmbed

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -68,8 +68,8 @@ <h1>Vbrick SDK Embed Test</h1>
 					<div>
 						<label for="embedType">Embed Type</label>
 						<div class="radio-grid">
-							<label><input type="radio" name="embedType" value="vod" checked>VOD</label><span></span>
-							<label><input type="radio" name="embedType" value="webcast">Webcast</label><span></span>
+							<label><input type="radio" name="embedType" id="embedType" value="vod" checked>VOD</label>
+							<label><input type="radio" name="embedType" id="embedType" value="webcast">Webcast</label>
```
Comment (lukeselden) [inline demo/index.html:72]: It isn't valid HTML to have two elements with the same id. the `form.elements` convention uses the name value, FWIW

**Review by lukeselden — APPROVED** [review]

File: demo/index.html (hunk 1 above)
Comment (ibirman-vbrick) [reply demo/index.html]: ok how about document.forms["demoform"].elements['embedType'].value == 'vod';

Comment (lukeselden) [reply demo/index.html]: your code `form.elements.embedType.value == 'vod'`  is fine already, though that suggestion works as well. I was just noting that you don't need the `id` parameters.

Comment (ibirman-vbrick) [reply demo/index.html]: Thanks, I'm all for setting a good example with our sdk demo

**Pull request merged**